src/main/java/pl/noip/lolstats/lol/stats/controller/RiotNameController.java

```diff
@@ -25,7 +28,7 @@ public RiotNameController(RiotRestClient riotRestClient) {
     public ResponseEntity<?> name(@RequestBody SummonerNameRequest summonerNameRequest) {
 
         List<String> userRegions = riotRestClient.findSummonersRegions(summonerNameRequest.getSumName());
-
+        log.info("there are some regions for that summoner name ");
```

Comment:
you can list how many regions using something like userRegions.size(). It's hint though, not change request.